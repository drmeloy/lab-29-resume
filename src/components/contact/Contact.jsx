import React from 'react';
import PropTypes from 'prop-types';

const Contact = ({ phone, email, sites }) => {
  const websites = sites.map((site, i) => (
    <li key={i}>{site}</li>
  ));

  return(
    <>
      <h2>Contact</h2>
      <p>{phone}</p>
      <p>{email}</p>
      <ul>{websites}</ul>
    </>
  );
};

Contact.propTypes = {
  phone: PropTypes.string.isRequired,
  email: PropTypes.string.isRequired,
  sites: PropTypes.array.isRequired
}

export default Contact;
