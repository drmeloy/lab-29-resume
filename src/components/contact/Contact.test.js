import React from 'react';
import { shallow } from 'enzyme';
import Contact from './Contact.jsx';

describe('Contact component', () => {
  it('renders a Contact component', () => {
    const wrapper = shallow(<Contact phone={''} email={''} sites={[]} />);
    expect(wrapper).toMatchSnapshot();
  })
});
