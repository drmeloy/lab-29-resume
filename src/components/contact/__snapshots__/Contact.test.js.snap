// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contact component renders a Contact component 1`] = `
<Fragment>
  <h2>
    Contact
  </h2>
  <p />
  <p />
  <ul />
</Fragment>
`;
